### DISCUSSION TITLE
What punctuation mark consists of a dot above a comma?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#87] [Galaxy Trivia] [answered]
**Hint:** . ,

**Comment by Tynab** [comment, marked as answer]
The punctuation mark that consists of a dot above a comma is called a "semicolon". The semicolon is a punctuation mark that is used to separate two closely related independent clauses in a sentence. It indicates a stronger separation than a comma, but a weaker separation than a period. The dot above the comma in a semicolon is sometimes called a "superior comma", and it is meant to indicate a slightly stronger pause or separation than a regular comma.
